src/pipeline-status-slack.py

```diff
@@ -200,6 +200,7 @@ def lambda_handler(event, context):
         "toggling_cost_saving_mode", False
     )
     slack_message = [f"*Execution:* <{execution_url}|{execution_name}>"]
+    attention = os.environ["ATTENTION"]
```

Comment:
Her må du JSON loade, ellers får du alltid true!
```suggestion
    attention = json.loads(os.environ["ATTENTION"])
```

Husk å sjekke at `json` er importert.

Reply:
Fiksa det på samme måten som den andre boolske miljøvariabelen.